lib/pf/web/httpd.collector_modperl_require.pl

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/perl
```

Comment:
It is used in 
https://github.com/jrouzierinverse/packetfence/blob/feature/mse-integration/conf/httpd.conf.d/httpd.collector.tt.example#L108